tools/integration/test/testConfig.js

```diff
@@ -21,6 +21,9 @@ const components = [
   'maven/mavencentral/org.apache.httpcomponents/httpcore/4.4.16',
   'maven/mavengoogle/android.arch.lifecycle/common/1.0.1',
   'maven/gradleplugin/io.github.lognet/grpc-spring-boot-starter-gradle-plugin/4.6.0',
+  'conda/conda-forge/-/numpy/1.13.0/',
+  'conda/conda-forge/linux-aarch64/numpy/-py36',
+  'conda/conda-forge/-/tensorflow',
   'crate/cratesio/-/ratatui/0.26.0',
```

Comment:
Thank you for sharing the test coordinates! The end-to-end tests will use these coordinates to fetch the harvest results and definitions. It is important to provide the complete coordinates. Out of the three packages, conda/conda-forge/linux-aarch64/numpy/1.16.6-py36hdc1b780_0 appears to be the smallest. This package might be the most suitable in terms of integration test speed.